test/index.js

```diff
@@ -466,4 +466,78 @@ describe('Hapi csv', () => {
             });
         });
     });
+
+    describe('Result key (e.g. for pagination)', () => {
+
+        it('Uses the result key', (done) => {
+
+            const result = {
+                items: [{
```

Comment:
Can you add a `page: 1` property in the result so it mimics the use case as close as possible?